torch/_functorch/partitioners.py

```diff
@@ -1780,8 +1780,14 @@ def get_recomputable_banned_nodes(
         ]
 
     recomputable_banned_nodes = get_recomputable_banned_nodes(banned_nodes)
-    # sort first by name, to ensure determinism when multiple nodes have same size
-    recomputable_banned_nodes = sorted(recomputable_banned_nodes, key=lambda x: x.name)
```

Comment:
This is not needed anymore as `banned_nodes` is now an `OrderedSet` https://github.com/pytorch/pytorch/pull/146102